### ISSUE TITLE
NVDA DOES NOT ANOUNCE ALL SHORTCUT KEYS FOR DROPDOWN MENUES IN OFFICE 365

### ISSUE THREAD

**Issue by netblue44** [nvaccess/nvda#7454]
In Spanish version of Office 365,
NVDA does not announce the file menu when ALT+A is pressed.
All other dropdown menues are anounced when there corresponding shortcut keys are used.
This does not happen with narrator.
I am using windows 10 creator Spanish

**Comment by Adriani90** [comment]
@netblue44 is this still an issue in NVDA 2019.1.1 and Office 365?

**Comment by Adriani90** [comment]
No updates from the initial author on this issue since years. I am closing as abandoned. Feel free to comment on this if you are still having this issue and we can reopen. Thanks.
